### DISCUSSION TITLE
Vercel integration: Can't get past the "https://sentry.io/extensions/vercel/link" screen

### DISCUSSION THREAD

**Discussion by benderillo** [getsentry/support#118] [Support]
**TL;DR;**

Trying to add Sentry integration to my project on Vercel, I can't get past the screen showing the page(https://sentry.io/extensions/vercel/link) and saying "Finish integration installation".

Clicking the "Install Vercel" button just reloads the page.

**Longer story:**

I had a hobby account with a project on Vercel that had integration with Sentry project.
I upgraded to Pro plan on Vercel but to my surprise it actually created another account and I had to transfer my project from the hobby to Pro account.
As part of the transfer my Sentry integration was not moved (they don't support it) so I have to re-integrate.

So I am trying now to connect my new project under Pro account on Vercel with my existing sentry project.
But the setup gets stuck on the screen where the Sentry page is loaded (https://sentry.io/extensions/vercel/link) saying "Finish Integration installation".
When I click the "Install Vercel" button, the page just reloads and nothing happens.

Not really sure where things break, Vercel or Sentry...
If anyone can help me by suggesting how do I debug this thing, that would be great!
